chocolatey/chocolatey-au.nuspec

```diff
@@ -10,6 +10,7 @@
     <summary>Chocolatey-AU is Powershell module that helps you to automate Chocolatey package updates</summary>
     <description></description>
     <projectUrl>https://github.com/chocolatey-community/chocolatey-au</projectUrl>
+    <iconUrl>https://chocolatey.org/assets/images/nupkg/chocolateyicon.png</iconUrl>

```

Comment:
Since this is under the chocolatey-community user, it probably should not be using the icon for official products? Similar to https://github.com/chocolatey-community/chocolatey-templates/issues/11 a new icon would probably need to be created.

Reply:
🤔 for some reason I thought I had taken this icon from the Chocolatey Community Validation Extension, but it looks like it's not using an icon either. Perhaps this one should leave the extension off for now.